### ISSUE TITLE
Unable to accept refer a friend invitation.

### ISSUE THREAD

**Issue by Aisu1324** [dalaranwow/dalaran-wow#5545]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: User is unable to accept a recruit a friend invitation and gets an error message.
**Expected behaviour**: User should be able to go through account creation with a friend invite.
**Steps to reproduce the problem**:

1. Recieve a recruit a friend invitation
2. Open the email with invitation
3. Click the link in the email
4. Proceed to continue sign up
5. Click on the "Create free account" button

![unable to connect](https://user-images.githubusercontent.com/41517290/43037502-b99cfe44-8cdb-11e8-8082-053074892a12.png)

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
